app.py

```diff
@@ -27,6 +27,11 @@ def create_movies_form():
 @app.post('/movies')
 def create_movie():
     # TODO: Feature 2
+    movie_name = request.form['movieName']
+    director_name = request.form['directorName']
+    rating = request.form['selectRating']
```

Comment:
Have you considered using request.form.get() so an exception isn't thrown if the name is ever not found?